server/core/store_test.go

```diff
@@ -92,3 +93,85 @@ func (s *testConcurrencySuite) TestCloneStore(c *C) {
 	}()
 	wg.Wait()
 }
+
+const (
+	MB = uint64(1024 * 1024)
```

Comment:
We already have some unit definitions in schedulers package

Reply:
it will cause cycle import